src/components/common/Accordion.js

```diff
@@ -5,25 +5,34 @@ import classNames from '../../utils/classNames'
 const Container = styled.div`
     overflow: hidden;
     height: 0;
-    transition: 250ms;
+    transition: ${props => props.transition}ms;
+    opacity: 0;
 
     &.open {
+        opacity: 1;
         height: ${props => props.height}px;
     }
 `
 
-const Accordion = ({ className, trigger, children }) => {
+const Accordion = ({ className, trigger, children, transition = '250' }) => {
     const [open, setOpen] = useState(false)
     const [contentHeight, setContentHeight] = useState('')
 
     useEffect(() => {
         const el = document.getElementById(trigger)
+        const secondEl = document.getElementById(`${trigger}-2`)
```

Comment:
@Patrick1904 nit: maybe it makes to do something for all events with some class vs having code explicitly only for 1 or 2 elements?